scylla-cql/src/types/deserialize/value.rs

```diff
@@ -3,7 +3,7 @@
 use bytes::Bytes;
 
 #[cfg(feature = "chrono")]
-use chrono::{NaiveDate, NaiveTime};
+use chrono::{DateTime, NaiveDate, NaiveTime, TimeZone as _, Utc};
```

Comment:
I'd remove the `use` clause from here. We already use full paths for `time` types. Let's be consistent and do the same for `chrono` types as well. This will be eventually removed once we decide to support other `chrono` versions.